src/elements/mission/mission.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Container, Divider, Typography, makeStyles } from '@material-ui/core'
+import { useTranslation } from 'react-i18next'
+
+const useStyles = makeStyles((theme) => ({
+  firstDivider: {
+    marginTop: theme.spacing(5)
+  },
+  secondDivider: {
+    marginTop: theme.spacing(0.5),
+    marginBottom: theme.spacing(6)
+  },
+  thirdDivider: {
+    marginTop: theme.spacing(6),
+    marginBottom: theme.spacing(0.5)
+  },
+  fourthDivider: {
+    marginBottom: theme.spacing(5)
+  },
+  textColor: {
+    color: theme.palette.primary.main
+  }
+}))
+
+export function Mission() {
+  const { t } = useTranslation()
```

Comment:
add a namespace here for now.
```suggestion
  const { t } = useTranslation('mission')
```